util/deploy.sh

```diff
@@ -59,7 +59,7 @@ then
 else
     QUIET_FLAG=""
 fi
-COMMAND="gcloud app deploy ${PROMOTE} ${QUIET_FLAG} --version=${VERSION} ${WPTD_PATH}/webapp"
+COMMAND="gcloud app deploy ${PROMOTE} ${QUIET_FLAG} --version=${VERSION} ${WPTD_PATH}/webapp ${WPTD_PATH}/results-processor"
```

Comment:
On second thought, let's make this deploy the passed-in path, which can be a required param.
`deploy.sh [-p] [-q] [-b] [-h] [app path]`

That way, we can have 2 different travis tasks and bypass them when nothing for that service has changed (see https://github.com/web-platform-tests/wpt.fyi/pull/122)

Reply:
I'm gonna back off my change in `deploy.sh` (i.e. not deploying `results-processor` to staging for now) in this PR. I'll fix the permission and change `deploy.sh` as you suggested in another PR.